src/routes/health.ts

```diff
@@ -0,0 +1,12 @@
+import { browser } from "../config";
+
+export async function handleHealthRequest(_req: Request): Promise<Response> {
+	return new Response(
+		JSON.stringify({
+			status: browser.isConnected() ? "ok" : "error",
```

Comment:
```suggestion
			status: browser.isConnected() ? "pass" : "fail",
```

https://datatracker.ietf.org/doc/html/draft-inadarei-api-health-check-06#section-3.1